main.go

```diff
@@ -57,7 +64,26 @@ var secretsName = flag.String(
 	"Secrets Name (version string of the helm chart)",
 )
 
+type envSettings map[string]string
+
+var templateEnv = make(envSettings)
+
+func (env *envSettings) String() string {
+	return ""
```

Comment:
I don't know when we would see this. And I have no idea how to represent a map value here. I'm happy to put in whatever you feel appropriate.

Given that the only caller of this program is a bash script under our control too, I'm not sure it matters much.

Reply:
I tried to return ` fmt.Sprintf("%v", env)`, but that shows a warning when running the tests:

```
./main.go:72:9: Sprintf format %v with arg env causes recursive String method call
```

I think returning the empty string is fine, given that the code will not be executed anyways.

Reply:
Sure, or just some hard coded `<env settings>` string or something; I'm mostly thinking _empty_ string would be confusing (since we wouldn't be able to troubleshoot).

Reply:
Fixed in c605f62